_posts/blog/2019-11-04-checkstyle-import-control.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Checking API use"
+---
+
+Accumulo follows [SemVer] across versions with the declaration of a public API.  Code not in the public API should be
+considered unstable, at risk of changing between versions.  The packages included in the public API are [listed on the website][api]
```

Comment:
```suggestion
considered unstable, at risk of changing between versions.  The public API packages API are [listed on the website][api]
```

Reply:
I assume you meant: "The public API packages are [listed on the website][api]"

Reply:
yes